### PR TITLE
update to setup instructions

### PR DESCRIPTION
**Pull request by bmanjones**
Added steps to use miniforge instead of the older miniconda instructions. Issue https://github.com/ocean-tracking-network/node-manager-training/issues/45

### CODE REVIEW

**Review by jdpye — APPROVED** [review]
good update by me.

File: _episodes/03_Setup_and_Installing_Needed_Software.md [hunk 1]
```diff
@@ -64,10 +79,11 @@ Then, OTN staff will give you access to the relevant Projects containing the cod
    * Activate the nodebook environment using `conda activate nodebook`
 1. You are now able to run commands in that folder. Now run: `git clone https://gitlab.oceantrack.org/otn-partner-nodes/ipython-utilities.git`. This will get the latest version iPython Utilities from our GitLab
 1. Navigate to the ipython-utilities subdirectory that was created by running `cd ipython-utilities`.
-1. Now to install all required python packages by running the following: `mamba env update -n nodebook -f environment.yml`
+1. Create a new conda environment by running the command: `conda create -n nodebook python=3.9`. This will create the environment named **nodebook** using the python version 3.9.XX.
+1. Now to install all required python packages by running the following: `mamba env update -n nodebook  t -f environment.yml`
 
 **To open and use the OTN Nodebooks:**
-- **MAC/WINDOWS**: Open your terminal, and navigate to your ipython-utilities directory, using `cd /paht/to/ipython-utilities`. Then, run the commands: 
+- **MAC/WINDOWS/LINUX**: Open your terminal, and navigate to your ipython-utilities directory, using `cd /paht/to/ipython-utilities`. Then, run the commands:
```
Comment (CaitlinBate) [inline _episodes/03_Setup_and_Installing_Needed_Software.md]: path is spelled wrong

File: _episodes/03_Setup_and_Installing_Needed_Software.md [hunk 2]
```diff
@@ -88,6 +104,10 @@ In the next lesson we will practice using our database console viewer and connec
 # More Useful Programs
 
 In order to work efficiently as a Node Manager, the following programs are necessary and/or useful.
+## Cross-Platform 
+
+**Visual Studio Code** - An advanced code editing integrated development environment (IDE).
+* https://code.visualstudio.com/
```
Comment (CaitlinBate) [inline _episodes/03_Setup_and_Installing_Needed_Software.md]: this is already down in line 125 - can we remove from one of the places? and to make the link render properly I think we need to do [link](link)

File: _episodes/03_Setup_and_Installing_Needed_Software.md [hunk 3]
```diff
@@ -27,9 +27,22 @@ This lesson will give attendees a chance to install all the relevant software, u
 
 `Mamba` is fast, cross-platform python distribution and a package manager. When you install `Mamba (through Miniforge)` you get a `python` interpreter, and many of the core python libraries. Managing your Python installation with Mamba allows you to be able to install and update all the packages needed to run the Nodebooks with one command rather than having to install each one individually. 
 
- **Install Miniforge3** - [https://conda-forge.org/miniforge/](https://conda-forge.org/miniforge/)
-  - Select the option to `add to PATH environment variable` (during install steps)!
-
+ **Miniforge Windows** - https://conda-forge.org/miniforge/ 
+  - Select the option install for Just Me (recommended).
+  - Check the option to Add Miniforge3 to my PATH environment variable.
```
Comment (CaitlinBate) [inline _episodes/03_Setup_and_Installing_Needed_Software.md]: make " Add Miniforge3 to my PATH environment variable." bold pls - this is an important step haha

File: _episodes/03_Setup_and_Installing_Needed_Software.md [hunk 4]
```diff
@@ -27,9 +27,22 @@ This lesson will give attendees a chance to install all the relevant software, u
 
 `Mamba` is fast, cross-platform python distribution and a package manager. When you install `Mamba (through Miniforge)` you get a `python` interpreter, and many of the core python libraries. Managing your Python installation with Mamba allows you to be able to install and update all the packages needed to run the Nodebooks with one command rather than having to install each one individually. 
 
- **Install Miniforge3** - [https://conda-forge.org/miniforge/](https://conda-forge.org/miniforge/)
-  - Select the option to `add to PATH environment variable` (during install steps)!
-
+ **Miniforge Windows** - https://conda-forge.org/miniforge/
```
Comment (CaitlinBate) [inline _episodes/03_Setup_and_Installing_Needed_Software.md]: i think the link needs to be [link](link) to make it clickable on the site

File: _episodes/03_Setup_and_Installing_Needed_Software.md [hunk 5]
```diff
@@ -27,9 +27,22 @@ This lesson will give attendees a chance to install all the relevant software, u
...
+  - Check the option to Add Miniforge3 to my PATH environment variable.
+
+ **Miniforge Mac** - 
+   - Setup homebrew by running the command: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` - **Note:** this operation requires elevated privileges (sudo)
+   - Use the commands outputted by brew to add brew to your system path
+   - Use brew to install miniforge: `brew install miniforge`
+   - Add miniforge to your zsh environment by typing conda init zsh. Restart the terminal
+
+**Miniforge Linux (Debian)**
+   - download Shell script (.sh) file from https://conda-forge.org/miniforge/
```
Comment (CaitlinBate) [inline _episodes/03_Setup_and_Installing_Needed_Software.md]: i think the link needs to be `[Words for Link](link url)`

**Review by CaitlinBate — CHANGES_REQUESTED** [review]

**Review by CaitlinBate — APPROVED** [review]

**Comment by CaitlinBate** [comment]
this can be merged once conflicts are inspected @bmanjones @jackVanish

**Pull request merged**